reana_client/cli/workflow.py

```diff
@@ -1144,15 +1144,17 @@ def workflow_run(
 )
 @click.option(
     "--include-workspace",
-    "workspace",
+    "should_delete_workspace",
     is_flag=True,
     help="Delete workspace from REANA.",
 )
```

Comment:
Should we remove this option? cc @tiborsimko 
If we keep it, we should change the help message to inform the user that this option does not change the behaviour of reana-client anymore

Reply:
I kept this option for backward compatibility. But, modifying the help message shouldn't be a problem.  I will do it.

Reply:
Yes, let's keep it for backwards compatibility, and document that as of REANA 0.9 release series it is forced on by default, that we always delete the workspace when we delete the workflow.

> In any case, how do we inform the user of this change?

We'll make a publicity for this breaking feature during reana-client 0.9 release. For new we can simply document it in CHANGES.rst. It is not necessary to do anything in the code, since we do want to flip the defaults (and the power users agree).